content/sensu-go/5.16/guides/scale-event-storage.md

```diff
@@ -174,8 +175,164 @@ To verify that the change was effective, look for messages similar to these in t
 Similar to enabling Postgres, switching back to the etcd datastore does not migrate current event data from one store to another.
 You may see old events in the web UI or sensuctl output until the etcd datastore catches up with the current state of your monitored infrastructure.
 
+## Configure Postgres streaming replication
+
+Postgres supports an active standby by using [streaming replication][6].
+All Sensu events written to the primary Postgres server will be replicated to the standby server.
+
+_**NOTE**: Paths and service names may vary based on your operating system._
+
+This section describes how to configure PostgreSQL streaming replication in four steps.
+
+### Step 1: Create and add the replication role
+
+If you have administrative access to Postgres, you can create the replication role:
+
+{{< highlight shell >}}
+$ sudo -u postgres psql
+postgres=# CREATE ROLE repl PASSWORD 'secret' LOGIN REPLICATION;
+CREATE ROLE
+postgres-# \q
+{{< /highlight >}}
+
+Then, you must add the replication role to `pg_hba.conf` using an [md5-encrypted password][5].
+Make a copy of the current `pg_hba.conf`:
+
+{{< highlight shell >}}
+sudo cp /var/lib/pgsql/data/pg_hba.conf /var/tmp/pg_hba.conf.bak
+{{< /highlight >}}
+
+Next, give the repl user permissions to replicate from the standby host.
+In the following command, replace `STANDBY_IP` with the IP address of your standby host:
+
+{{< highlight shell >}}
+export STANDBY_IP=192.168.52.10
+echo "host replication repl ${STANDYB_IP}/32 md5" | sudo tee -a /var/lib/pgsql/data/pg_hba.conf
+{{< /highlight >}}
+
+Restart the PostgreSQL service to activate the `pg_hba.conf` changes:
+
+{{< highlight shell >}}
+sudo systemctl restart postgresql
+{{< /highlight >}}
+
+### Step 2: Set streaming replication configuration parameters
+
+The next step is to set the streaming replication configuration parameters on the primary Postgres host.
+Begin by making a copy of the `postgresql.conf`:
+
+{{< highlight shell >}}
+sudo cp -a /var/lib/pgsql/data/postgresql.conf /var/lib/pgsql/data/postgresql.conf.bak
+{{< /highlight >}}
+
+Next, append the necessary configuration options.
+
+_**NOTE**: To enable read-only queries on a standby server, you must set `wal_level` to `hot_standby`.
```

Comment:
Sensu does not provide means to make use of read-only replicas, so I think we should remove this note.